### PR TITLE
KillFeed mod 

### PR DESCRIPTION
**Pull request by Frooszy**
Tasks:
- [X] Base of mod

User Interface: 
- [X] Distance on KillFeed
- [X] Add Team colors on UI
- [X] KillFeed final corrections 

Functionalities:

- [X] Final corrections

### CODE REVIEW

**Comment by Frooszy** [comment]
The interface part is already fully completed, now just finish the part of detecting various ways for a player to kill another.

File: @killfeed/src/gui/layouts/muderinfoextra_gui.layout
Comment (TiagoDamascena) [inline @killfeed/src/gui/layouts/muderinfoextra_gui.layout:1]: Where this file is beeing used? Cant find any reference to it on the code

File: scripts/client.bat
Comment (TiagoDamascena) [inline scripts/client.bat:1]: You forgot to remove the -name option from the client

**Review by TiagoDamascena — COMMENTED** [review]
Please, review the indentation of the code, some places are using 4 spaces (the correct indentation) but others are using tabs. In general the quality of the code is very good, just fix those minor details and i will merge this branch.

**Review by TiagoDamascena — APPROVED** [review]

**Pull request merged**